docs/getting-started/Typed_Linting.mdx

```diff
@@ -209,11 +264,6 @@ This means that generally they usually only run a complete lint before a push, o
 
 **We strongly recommend you do use type-aware linting**, but the above information is included so that you can make your own, informed decision.
 
-### I get errors telling me "The file must be included in at least one of the projects provided"
-
-You're using an outdated version of `@typescript-eslint/parser`.
-Update to the latest version to see a more informative version of this error message, explained in our [Troubleshooting and FAQs page](../troubleshooting/FAQ.mdx#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file).
-
```

Comment:
This hasn't been the error message format for quite a while, and closely matches the one mentioned in the frequently-linked-to troubleshooting docs. I feel comfortable removing this legacy affordance to take up less space on the page.